config/samples/source_v1alpha1_gitrepository.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: source.fluxcd.io/v1alpha1
 kind: GitRepository
 metadata:
-  name: podinfo
+  name: gitrepository-sample
```

Comment:
This change breaks CI, we should update the smoke test if Kubebuilder generates the sample name.